src/patternfly/layouts/Gallery/gallery.hbs

```diff
@@ -1,3 +1,3 @@
-<div class="pf-l-gallery {{ modifierClass }}">
+<div class="pf-l-gallery{{#if gallery--modifier}} {{gallery--modifier}}{{/if}}">
```

Comment:
add a `gallery--attribute` block.

Reply:
done

Reply:
have you pushed this to the branch? still not seeing it.

Reply:
not seeing that either